decoupled-site/frontend/src/graphql/Search.graphql

```diff
@@ -0,0 +1,23 @@
+query Search($searchParam: String, $locales: Locales, $order: OrderBy) {
```

Comment:
Can you set an empty string as default so we always get everything returned in case there is no query.

For example, here I just clicked on the search button without a query. Instead of an empty list, it is better for SEO to return everything.

![Screenshot 2023-04-24 at 12 38 47](https://user-images.githubusercontent.com/1325546/233973889-4996c719-712c-4dca-a2c3-5c752760a3f6.png)